style.scss

```diff
@@ -44,6 +49,7 @@ body {
 .icon {
     width: 50px;
     height: 50px;
+    margin: 5px;
```

Comment:
It's a good point to add offsets between icons. It would be better to add this margin not to the icon inside the link, but to the link element itself.